bin/vault_utils/engine.py

```diff
@@ -38,10 +38,10 @@ def engine_at_path(cls, vault, engine_type, engine_path):
 
         return engine_class(vault, engine_path)
 
-    def _get(self, path):
+    def _get(self, path, params={}):
```

Comment:
Style question... Do we want params as an explicit argument, or do we want params as a kwargs argument e.g. `**params`